bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -31,6 +34,18 @@ class Edge extends WebBrowser {
 		Library.loadLibrary("WebView2Loader");
 	}
 
+	static {
+		URI absolutePath;
+		try {
+			Path tempFile = Files.createTempFile("base", ".html");
+			absolutePath = tempFile.toUri();
+			tempFile.toFile().deleteOnExit();
+		} catch (IOException e) {
+			absolutePath = URI.create("about:blank");
+		}
+		BASE_URI = absolutePath;
```

Comment:
Are there reasons for writing the value to a temporary local variable instead of directly assigning the value to field?

Reply:
![image](https://github.com/user-attachments/assets/e8f41602-0181-4c78-b981-40b836df0450)
Asks to remove final